### PR TITLE
merge CL:0002294 into CL:0009077

### LINKED ISSUE #2101: CL:0009077 & CL:0002294 are duplicates?
CL:0002294 (type-1 epithelial cell of thymus)  has exactsynonym subcapsular thymic epithelial cell which is the rdfs:label of CL:0009077. I am guessing they should be merged? Also I thought there was suppose to be a QC check that prevents this @anitacaron or @gouttegd?

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #2101 
Requires changes in Uberon (changing UBERON:0006936 to use CL:0009077)
My choice of choosing CL:0009077 was due to its use in HRA subset 
I see that there are obsoletion notices now - not sure exact ways of working now so will leave that to @bvarner-ebi 
Also due to obsoletion method, import file is changed - happy for this change to be reverted and fixed at source (Uberon). 
PS could someone help me fix it in [Uberon side please](https://github.com/obophenotype/uberon/issues/3024)? requires import and its a whole thing here. 

Thanks!

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
@shawntanzk, thanks for the suggested changes. 
If you propose to merge/obsolete a term, kindly just open an issue. An editor will review and make an announcement on slack and give at least 2 weeks' notice before obsoleting if there is no objection. We're requesting outside collaborators to request PR reviewers instead of automatically assigning them.

**Comment by shawntanzk** [comment]
@bvarner-ebi - happy for you to close this and go through your processes :) thanks

**Comment by shawntanzk** [comment]
> We're requesting outside collaborators to request PR reviewers instead of automatically assigning them.

btws @bvarner-ebi  sorry for assigning directly, I figured that was the way to request PR reviewers. Do let me know how I should go about it in the future. Personally I dont really mind who reviews my PRs, but I tend to find that if I don't assign a reviewer, the PR sits there and nothing happens. Is there a system to pick up PR reviews in CL if I dont really mind who reviews my PR?

Currently I am following CL documentation to assign a reviewer https://obophenotype.github.io/cell-ontology/contributing/#pull-request-guidelines quoting the docs:

> Make sure pull requests have someone assigned to review them and remind them once in a while. Do not let them go dormant

Happy to follow whatever procedure is preferred by the community, but would like it documented cause different ontologies have different preferences in workflow and all :) 

PS on a seperate note, I notice that @gouttegd was to be generic assignee for tickets in the documentation lol - I think this was written up when he was sheparding CL tickets, might consider a change?

File: src/ontology/imports/merged_import.owl
Comment (anitacaron) [inline src/ontology/imports/merged_import.owl:1]: why do you have changes in the `merged_import.owl`?

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/imports/merged_import.owl
Comment (shawntanzk) [reply src/ontology/imports/merged_import.owl:1]: can revert them - happen the term to be obsolete has a uberon term in there, and so obsoletion in protege removes/change all axioms, one (or more) of which comes from import. I have opened another ticket for uberon already, so better way would probably be to let this be and make sure uberon side changes, gets imported into cl, before release - of course with the 2 weeks notification, this has got to be coordinated and all, so will leave that to you all. For now I will revert changes in merged_import

**Comment by bvarner-ebi** [comment]
This PR involves obsoleting a CL class.

Will close without merging for now ([acknowledged by assignee](https://github.com/obophenotype/cell-ontology/pull/2110#issuecomment-1683498928)) with following action items:

- When time permits, editor will [review ticket](https://github.com/obophenotype/cell-ontology/issues/2101), if necessary find literature support to justify change
- notify slack #cl-editors of planned obsoletion 
- if no objection after at least 2 weeks, proceed with merge
- next CL release is planned for next week, so if there are no objections change will be reflected in September release at the earliest
- work with @anitacaron to incorporate synonym QC in CL checks

@shawntanzk, thanks again for spotting these potential changes to improve CL.

**Pull request closed without merging**
